app/controllers/foods_controller.rb

```diff
@@ -1,21 +1,19 @@
 class FoodsController < ApplicationController
-  def create; end
-
-  def create_food_linked_with_inventory
-    @inventory = Inventory.find(params[:inventory_id])
+  def new
```

Comment:
You can include the required methods for the view
After which, a user can directly borrow from this controller-view to add a new food based on the recipe creaded